plugins/agent_types/AcquiaLiftLearn.inc

```diff
@@ -80,22 +80,25 @@ class AcquiaLiftLearn extends PersonalizeAgentBase implements AcquiaLiftLearning
   public function getAgentSyncOperations($option_sets, $new_goals, $old_goals = array()) {
     // Currently only a single option set is supported.
     $option_set = reset($option_sets);
+    // If the test agent has a site name, use this as a prefix for goals and
+    // decision sets.
+    $prefix = isset($this->data['site_name']) ? $this->data['site_name'] . '-' : '';
```

Comment:
It's probably not important at all... just the only thing that caught my eye and made me think it would be nice to just call acquia_lift_get_site_prefix($agent_data) (or something).  So NOT a blocker for this patch going through.